SPECS/binutils/CVE-2025-8224.patch

```diff
@@ -0,0 +1,27 @@
+From c51657d07c6ee1f609700a8f39be29ba7fdeb65e Mon Sep 17 00:00:00 2001
+From: Azure Linux Security Servicing Account
+ <azurelinux-security@microsoft.com>
+Date: Mon, 28 Jul 2025 19:05:31 +0000
+Subject: [PATCH] Fix CVE CVE-2025-8224 in binutils
+
+[AI Backported] Upstream Patch Reference: https://sourceware.org/git/gitweb.cgi?p=binutils-gdb.git;h=db856d41004301b3a56438efd957ef5cabb91530
+---
+ bfd/elf.c | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/bfd/elf.c b/bfd/elf.c
+index 1de0cec0..e49d26fd 100644
+--- a/bfd/elf.c
++++ b/bfd/elf.c
+@@ -287,7 +287,7 @@ bfd_elf_get_str_section (bfd *abfd, unsigned int shindex)
+ 
+       /* Allocate and clear an extra byte at the end, to prevent crashes
```

Comment:
As per the upstream patch, update patch to also remove the stale comments:
[Reference](https://gitlab.com/gnutools/binutils-gdb/-/commit/db856d41004301b3a56438efd957ef5cabb91530#569b4d3c1d9424acf76b2396f481336184a07562_288_288)
`* elf.c (bfd_elf_get_str_section): Remove outdated comment, and
	tweak shstrtabsize test to suit.`